dependencies/influxdb/http.go

```diff
@@ -65,6 +68,25 @@ func (h HttpProvider) SeriesCardinalityReaderFor(ctx context.Context, conf Confi
 	}, nil
 }
 
+func (h HttpProvider) WriterFor(ctx context.Context, conf Config) (PointsWriter, error) {
+	httpClient, err := h.clientFor(ctx, conf)
+	if err != nil {
+		return nil, err
+	}
+	idThenName := func (n NameOrID) string {
```

Comment:
Can you make this a global function so it can be called in other methods potentially?

The primary reason this package doesn't use the influxdb go client is because there was a v0 version being used in one project that depends on flux and there was a newer v1 and they had different APIs so we weren't able to use the official API. This package should be using the official API everywhere if it's available and I have a feeling this functionality will be needed for the query methods too.